common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1214 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: amqstreams.v1.4.0
+  namespace: placeholder
+  annotations:
+    capabilities: Deep Insights
+    alm-examples: |-
+      [  
+        {  
+            "apiVersion":"kafka.strimzi.io/v1beta1",
+            "kind":"Kafka",
+            "metadata":{  
+              "name":"my-cluster"
+            },
+            "spec":{  
+              "kafka":{  
+                  "version":"2.4.0",
+                  "replicas":3,
+                  "listeners":{  
+                    "plain":{  
+
+                    },
+                    "tls":{  
+
+                    }
+                  },
+                  "config":{  
+                    "offsets.topic.replication.factor":3,
+                    "transaction.state.log.replication.factor":3,
+                    "transaction.state.log.min.isr":2,
+                    "log.message.format.version":"2.4"
+                  },
+                  "storage":{  
+                    "type":"ephemeral"
+                  }
+              },
+              "zookeeper":{  
+                  "replicas":3,
+                  "storage":{  
+                    "type":"ephemeral"
+                  }
+              },
+              "entityOperator":{  
+                  "topicOperator":{  
+
```

Comment:
Do we need the empty line here and in UO?

Reply:
I don't think so but it's in this way since 1.2.0 and even for other fields.